### ISSUE TITLE
[Obsolete] CL:1000606 'kidney nerve cell'

### ISSUE THREAD

**Issue by paolaroncaglia** [obophenotype/cell-ontology#1067]
**CL term**
CL:1000606 'kidney nerve cell'

**Reason for deprecation**
Equivalent to neuron and ('part of' some kidney)
subclass of 'kidney interstitial cell'
Not defined. The only annotation is KUPO:0001015. I suspect that the intent was to represent neuron projections that reach kidneys. My understanding is that you'd only use 'part of' x if the neuron soma was located in x (and I know CL has more specific axioms for that). Should the term be obsoleted?  

**Suggested replacement to consider**
An anatomy term for renal nerves? Haven't had time to look into one.

**Comment by nicolevasilevsky** [comment]
I'm okay with obsoleting this too

**Comment by paolaroncaglia** [comment]
If useful, ontologies that use CL:1000606 'kidney nerve cell' according to Ontobee [here](http://www.ontobee.org/ontology/CL?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_1000606). Thanks!

**Comment by dosumis** [comment]
I agree it should be obsoleted.  Do we know anyone at KUPO we could ask about usage?

**Comment by paolaroncaglia** [comment]
@cmungall (@uberon ) please

> I agree it should be obsoleted. Do we know anyone at KUPO we could ask about usage?

CL:1000606 'kidney nerve cell' isn't used in GO. It has no informative annotation, other than the KUPO dbxref.
Based on the ID range, it was created by @cmungall . Chris, can we safely assume that this grouping term is ok to keep? Thank you!

**Comment by paolaroncaglia** [comment]
I will close this ticket and won't obsolete CL:1000606 'kidney nerve cell' because

- it was created by Chris with a KUPO dbxref, so there might have been specific reasons to create it
- I've looked into an in-depth [paper](https://cjasn.asnjournals.org/content/10/10/1831) on the anatomical entity that CL:1000606 is said to be located in (a paper brought to attention as part of another [ticket](https://github.com/obophenotype/cell-ontology/issues/1528#issuecomment-1088500630)). While it doesn't explicitly mention neurons, it does mention nerve endings.
